drgn/commands/_builtin/crash/_bpf.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+# ebpf-related commands.
+
+import argparse
+from typing import Any, List, Sequence
+
+from drgn import Program
+from drgn.commands.crash import crash_command
+from drgn.helpers.common.format import CellFormat, print_table
+from drgn.helpers.linux.bpf import (
+    bpf_map_for_each,
+    bpf_prog_for_each,
+    bpf_prog_subprogs,
+    bpf_prog_trampoline_progs,
+    bpf_prog_used_maps,
+)
+
+
+@crash_command(
+    description="display all eBPF programs and maps",
+)
+def _crash_cmd_bpf(
+    prog: Program, name: str, args: argparse.Namespace, **kwargs: Any
+) -> None:
+    prog_rows: List[Sequence[Any]] = [
+        (
+            "ID",
```

Comment:
It looks like crash centers this heading:


```suggestion
            CellFormat("ID", "^"),
```